### PR TITLE
OMAP: Create exemple Structure file for OMAP4/5 PandaBoardES.

### PR DESCRIPTION
**Pull request by sguiriec**
This patch is adding reference structure file for OMAP4/5 Panda.
It is including the reference Structure file. Some additional
work is needed to create the different PFW XML file for routing.
Files has been tested on both PandaBoardES and PandaBoard5 (uevm)

TODO: Create scritp for ALSA playback/capture with PFW criteria.

Signed-off-by: Sebastien Guirec <sebastien.guiriec@intel.com>

### CODE REVIEW

**Comment by sguiriec** [comment]
@dawagner @Makohoek @gdenneul Please review

**Comment by Makohoek** [comment]
Some minor reworks are necessary before merge :-1:

**Comment by sguiriec** [comment]
Branch updated according to comments and all the patches squash into a single one. 
@dawagner @Makohoek @gdenneul Please review

File: omap-tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+The PFW test has been done with UBUNTU Core File System or Poky minimal image..
+
+- File System:
+-------------
+Download ubuntu core FS.
+=> Untar the FS on the File system partition
+
+* Remove Root password
+sudo gedit /media/rootfs/etc/shadow
```
Comment (dawagner) [inline omap-tutorial/README.md]: This does not render well. Check the "rich diff" or check here: https://github.com/01org/parameter-framework-samples/blob/sgc/topic/pfw_omap/omap-tutorial/README.md

File: omap-tutorial/README.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+ # Use the path set during Cmake build of PFW core.
+export LD_LIBRARY_PATH=/usr/local/lib
+
+test-platform ./PandaBoardES.xml
+remote-process localhost 5001 start
+remote-process localhost 5000 listParameters /Audio
+remote-process localhost 5000 help
+```
+Exemple for Headset Playback on Media port.
+'''
```
Comment (dawagner) [inline omap-tutorial/README.md]: these should be backticks.

File: omap-tutorial/README.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+ # Configure ABE/AESS
+remote-process localhost 5000 setParameter /Audio/omapabe/abe/output/headset/dl1/media_playback 120
+remote-process localhost 5000 setParameter /Audio/omapabe/abe/output/headset/dl1/mixer_media 1
+remote-process localhost 5000 setParameter /Audio/omapabe/abe/output/headset/sdt_dl 1
+remote-process localhost 5000 setParameter /Audio/omapabe/abe/output/headset/sdt_dl_volume/volume 120
+remote-process localhost 5000 setParameter /Audio/omapabe/abe/output/pdm 1
+ # Test playback
+aplay -Dplughw:0,0 test.wav
+
+'''
```
Comment (dawagner) [inline omap-tutorial/README.md]: and these too.

**Comment by dawagner** [comment]
Just some markdown issues.

**Comment by sguiriec** [comment]
Fix UBUNTU root password removal inside README.md for UBUNTU FS creation.
@dawagner @Makohoek Please review

**Comment by dawagner** [comment]
Thanks :)

**Pull request merged**